Plan/velocity/build.gradle

```diff
@@ -24,6 +24,5 @@ dependencies {
 }
 
 shadowJar {
-    dependsOn blossomSourceReplacementJava
```

Comment:
Is this now handled by blossom?

Reply:
```java
// Blossom.java
 this.project.getTasks().named(compileTaskName, AbstractCompile.class, task -> {
      task.dependsOn(sourceReplacementTask);
```
![image](https://user-images.githubusercontent.com/13612979/123234504-aa9cce80-d4e3-11eb-947a-27910816dfa9.png)

Blossom automatically adds a `dependsOn()` to the `compileJava` task of the project on build configure, so yes, Blossom handles it. Our Sponge project doesn't even have this line 😄

Reply:
Alright, thanks for the fix :)